internal/server/models/access_key.go

```diff
@@ -20,9 +20,9 @@ type AccessKey struct {
 	OrganizationMember
 	Name string `gorm:"uniqueIndex:idx_access_keys_name,where:deleted_at is NULL"`
 	// IssuedFor is the ID of the user that this access key was created for
-	IssuedFor         uid.ID
-	IssuedForIdentity *Identity `gorm:"foreignKey:IssuedFor" db:"-"`
-	ProviderID        uid.ID
+	IssuedFor     uid.ID
+	IssuedForName string
```

Comment:
We only use the name for this join, so at least for now I changed this field to only the name of the identity.

When we've finished converting the identity queries over we could go back to the full struct, but I don't see a reason to do that until we make use of more of the fields.